alexrallen/sonic-buildimage
# [mellanox] remove 2x40G and 4x40G breakout modes due to no hardware support

#### Why I did it

Mellanox platforms do not support 2x40G or 4x40G breakout modes. 

#### How I did it

Removed these breakout modes from platform.json and hwsku.json files 

#### How to verify it

Run `show interface breakout` and verify no 2x40G or 4x40G modes are shown in the supported breakout modes. 

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog
[mellanox] remove 2x40G and 4x40G breakout modes due to no hardware support


#### A picture of a cute animal (not mandatory but encouraged)
![donkey](https://user-images.githubusercontent.com/5898707/127019573-9979c2aa-24c1-499d-b699-fa839de4ffa2.jpg)
